src/com/mostc/pftt/model/core/PhpBuild.java

```diff
@@ -58,7 +58,7 @@ public PhpBuild(String build_path) {
 	 * @return
 	 */
 	public boolean checkOpenSSLVersion(ConsoleManager cm, Host host, String str) {
-		return is53(cm, host)||is54(cm, host)?str.contains("0.9.8"):str.contains("1.0.1");
+		return str.contains("1.0.1");
```

Comment:
1.0.1 isn't even used anywhere?

Thanks.

Reply:
No, it is not. I didn't touch any of *this* out-dated stuff in this PR. For instance, https://github.com/php/pftt2/pull/35/files#diff-cdc00e4b20eb5d0966c9457a1b7f2b8d is referring to "Apache2416-VC14-OpenSSL1.0.1e-x64" and such. I think fixing this should be part of another PR.

Reply:
Ok, makes sense.

Thanks.